lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -237,7 +237,7 @@ def define
 
       outcome :outcome_opposite_sex_marriage_poland_residing_in_uk_or_ceremony_country
 
-      outcome :outcome_os_slovenia
+      outcome :outcome_opposite_sex_marriage_slovenia_residing_in_uk_or_ceremony_country
```

Comment:
I've renamed the outcome to `outcome_opposite_sex_marriage_in_slovenia_when_residing_in_uk_or_slovenia ` in c797a144c7c21d8212bcee0fcd0a4288d55292c5, which is in PR #2409.